tests/pyfunc/test_dependencies_functions.py

```diff
@@ -110,3 +111,81 @@ def mock_get_installed_version_fn(package, module=None):
                     "detecting model dependency mismatches"
                 )
             )
+
+
+@pytest.mark.large
+def test_get_model_dependencies(tmpdir):
```

Comment:
```suggestion
def test_get_model_dependencies(tmp_path):
```

Let's use the `tmp_path` fixture.